.github/workflows/develop_loaderweb.yml

```
@@ -0,0 +1,65 @@
+    runs-on: windows-latest
+    needs: build
+    permissions:
+      id-token: write #This is required for requesting the JWT
+      contents: read #This is required for actions/checkout
+
+    steps:
+      - name: Download artifact from build job
+        uses: actions/download-artifact@v4
+        with:
+          name: .net-app
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)